sympy/stats/joint_rv_types.py

```diff
@@ -0,0 +1,228 @@
+
+    _argnames = ['mu', 'sigma']
+    is_Continuous=True
+
+    @property
+    def set(self):
+        k = len(self.mu)
+        return S.Reals**k
+
+    def check(self, mu, sigma):
+        mu, sigma = Matrix([mu]), Matrix(sigma)
+        _value_check(len(mu) == len(sigma.col(0)),
+            "Size of the mean vector and covariance matrix are incorrect.")
+        #check if covariance matrix is positive definite or not.
+        _value_check(all([i > 0 for i in sigma.eigenvals().keys()]),
+            "The covariance matrix must be positive definite. ")
+
+    def pdf(self, *args):
+        from sympy.functions.special.bessel import besselk
+        mu, sigma = Matrix(self.mu), Matrix(self.sigma)
+        k = len(mu)
+        sigma_inv = sigma**(-1)
+        args = Matrix(args).transpose()
+        v = 1 - S(k)/2
+        return S(2)/((2*pi)**(S(k)/2)*sqrt(det(sigma))) \
+        *((args.transpose()*sigma_inv*args)[0]/(2 + (mu.transpose()*sigma_inv*mu)[0])) \
+        **(S(v)/2)*besselk(v, sqrt((2 + (mu.transpose()*sigma_inv*mu)[0])
+            *((args.transpose()*sigma_inv*args)[0])))\
+        *exp((args.transpose()*sigma_inv*mu)[0])
+
+def MultivariateLaplace(syms, mu, sigma):
+    """
+    Creates a joint random variable with multivariate laplace distribution.
+
+    Parameters:
+    ==========
+
+    syms: list/tuple/set of symbols for identifying each component
+    mean_vector: A list/tuple/set consisting of k means,represents a k
+        dimensional mean vector
+    covariance_mat: The covariance matrix for the distribution
+
+    Returns:
+    =======
+
+    A random symbol
+    """
+    return rv(MultivariateLaplaceDistribution, syms, mu, sigma)
+
+#-------------------------------------------------------------------------------
+# Multivariate T-distribution ---------------------------------------------------------
+
+class MultivariateTDistribution(JointDistribution):
+
+    _argnames = ['mu', 'shape_mat', 'dof']
+    is_Continuous=True
+
+    @property
+    def set(self):
+        k = len(self.mu)
+        return S.Reals**k
+
+    def check(self, mu, sigma, v):
+        mu, sigma = Matrix([mu]), Matrix(sigma)
+        _value_check(len(mu) == len(sigma.col(0)),
+            "Size of the location vector and shape matrix are incorrect.")
+        #check if covariance matrix is positive definite or not.
+        _value_check(all([i > 0 for i in sigma.eigenvals().keys()]),
+            "The shape matrix must be positive definite. ")
+
+    def pdf(self, *args):
+        from sympy.functions.special.gamma_functions import gamma
+        mu, sigma = Matrix(self.mu), Matrix(self.shape_mat)
+        v = S(self.dof)
+        k = S(len(mu))
+        sigma_inv = sigma**(-1)
+        args = Matrix(args)
+        return gamma((k + v)/2)/(gamma(v/2)*(v*pi)**(k/2)*sqrt(det(sigma)))\
+        *(1 + 1/v*((args - mu).transpose()*sigma_inv*(args - mu))[0])**((-v - k)/2)
+
+def MultivariateT(syms, mu, sigma, v):
```

Comment:
maybe code for another PR

Reply:
Made the changes. I'll send the seperate PR after this is merged.